tests/components/rest_command/test_init.py

```diff
@@ -115,8 +115,11 @@ def test_rest_command_timeout(self, aioclient_mock):
 
         aioclient_mock.get(self.url, exc=asyncio.TimeoutError())
 
-        self.hass.services.call(rc.DOMAIN, "get_test", {})
-        self.hass.block_till_done()
+        with pytest.raises(
+            HomeAssistantError,
+            match=r"^Timeout when calling resource 'https://example.com/'$",
+        ):
+            self.hass.services.call(rc.DOMAIN, "get_test", {}, blocking=True)
```

Comment:
Side note: This is one of the few test modules that still uses sync tests.

Reply:
Oh indeed, gonna refactor this